### PR TITLE
✨ S4L Lite: 11.08 meeting feedback (⚠️ devops)

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Rename product: s4llight -> s4llite
- [x] Login Page
  - Shorter hash
  - Company name
  - New splitimage
  - License section in Cookie Policy dialog
- [x] Feedback via email
- [x] Forum link

Devops: rename product name (I'll take care of it)

image
image

## Related issue/s

related to https://github.com/ITISFoundation/osparc-issues/issues/740
related to https://github.com/ITISFoundation/osparc-issues/issues/765

### CODE REVIEW

**Review by pcrespov — CHANGES_REQUESTED** [review]
This image ``services/static-webserver/client/source/resource/osparc/s4llite_splitimage.png`` is over 1MB limit per file in our repo.
The pre-commit hooks should have stop this commit. Does it work in your computer?

Is there any chance to move these images to another repo and use links to them?

File: services/static-webserver/client/source/class/osparc/navigation/Manuals.js [hunk 1]
```diff
@@ -53,18 +53,101 @@ qx.Class.define("osparc.navigation.Manuals", {
       return manuals;
     },
 
+    __openGithubIssueInfoDialog: function() {
+      const issueConfirmationWindow = new osparc.ui.window.Dialog("Information", null,
+        qx.locale.Manager.tr("To create an issue in GitHub, you must have an account in GitHub and be already logged-in.")
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/navigation/Manuals.js:58]: Q: have you ever tried to activate the translation?

**Review by pcrespov — COMMENTED** [review]
Add also a note

**Comment by mguidon** [comment]
Is this the right email address? Or should that go to zmt support?
We already add the zmt forum I guess.

File: services/static-webserver/client/source/class/osparc/navigation/Manuals.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/navigation/Manuals.js:58]: 4 years ago... then we dropped it, but the frontend fully supports it.

Comment (pcrespov) [reply services/static-webserver/client/source/class/osparc/navigation/Manuals.js:58]: i think it is a shame. Perhaps we should bring that up again in the s4l meeting?

**Comment by odeimaiz** [comment]
> * This image `services/static-webserver/client/source/resource/osparc/s4llite_splitimage.png` is over 1MB limit per file in our repo.
>   The pre-commit hooks should have stop this commit. Does it work in your computer? See https://github.com/ITISFoundation/osparc-simcore/blob/9dfda536dbf980013762c2a242afce1297d525e2/.pre-commit-config.yaml#L10
> 
> Is there any chance to move these images to another repo and use links to them?

I reduced the file size.

**Comment by odeimaiz** [comment]
> Is this the right email address? Or should that go to zmt support? We already add the zmt forum I guess.

All that information goes in the DB

**Review by pcrespov — APPROVED** [review]
cool!


Q: If we had a content server deployed (as https://github.com/ITISFoundation/osparc-issues/issues/702). We would not need to add these files here, right? Just to be sure I understand it right.

File: services/static-webserver/client/source/class/osparc/navigation/Manuals.js [hunk 2]
```diff
@@ -139,10 +139,16 @@ qx.Class.define("osparc.navigation.Manuals", {
         menu.add(newFogbugzIssueBtn);
       }
 
-      if ((statics.s4lSupportEmail || statics.s4lliteSupportEmail) && (osparc.utils.Utils.isProduct("s4l") || osparc.utils.Utils.isProduct("s4llite"))) {
-        const giveFeedbackBtn = new qx.ui.menu.Button(qx.locale.Manager.tr("Give us Feedback"));
-        giveFeedbackBtn.addListener("execute", () => this.__openSendEmailFeedbackDialog(statics), this);
-        menu.add(giveFeedbackBtn);
+      if (osparc.utils.Utils.isProduct("s4l") || osparc.utils.Utils.isProduct("s4llite")) {
+        const forumBtn = new qx.ui.menu.Button(qx.locale.Manager.tr("S4L Forum"));
+        forumBtn.addListener("execute", () => window.open("https://forum.zmt.swiss/"), this);
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/navigation/Manuals.js:144]: we could add a new link in the product's table for this as well

Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/navigation/Manuals.js:144]: image

**Comment by odeimaiz** [comment]
> cool!
> 
> Q: If we had a content server deployed (as [ITISFoundation/osparc-issues#702](https://github.com/ITISFoundation/osparc-issues/issues/702)). We would not need to add these files here, right? Just to be sure I understand it right.
> 
> Do not forget to add (⚠️ devops) mark in the title with instructions to change the product config to the new name.

Correct

**Review by GitHK — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]
looks lite

**Pull request merged**